common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1023,6 +1023,16 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    enable_discussion_flag = Boolean(
```

Comment:
Please remove the trailing `flag`. That is not a convention used here. Please follow the convention of `xyz_enabled` as is used for other boolean values. i.e. rename this to `enable_discussions`.